src/components/WelcomeView.jsx

```diff
@@ -110,7 +107,12 @@ function WelcomeView({ terraUrl }) {
               target="_blank"
               rel="noopener noreferrer"
             >
-              <JadeLink>
+              <JadeLink
+                sx={{
+                  paddingTop: theme.spacing(2),
+                  paddingBottom: theme.spacing(2),
+                }}
```

Comment:
```suggestion
              <JadeLink
                sx={{
                  paddingTop: 2,
                  paddingBottom: 2,
                }}
```
We can use the sx shorthand here  (and then we do remove the reliance on useTheme, but I don't think we should need that anyway)